src/tests/jobtest.c

```diff
@@ -30,26 +30,130 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <strings.h>
 #include <daos.h>
+#include <daos_mgmt.h>
+#include <daos/common.h>
+#include <daos/pool.h>
+
+static char *progname;
+
+void print_usage()
+{
+	fprintf(stderr, "Usage: %s -p pool_str [-s nsecs] [-x] \
+		[-h handles_per_pool]\n", progname);
+}
+
+void cleanup_handles(uuid_t *pool_uuids, int num_pools,
+			daos_handle_t **pool_handles, int handles_per_pool)
+{
+	int i, rc;
+
+	for (i = 0; i < num_pools; i++) {
+		int idx;
+		if (uuid_is_null(pool_uuids[i])) {
+			continue;
+		}
+		for ( idx = 0; idx < handles_per_pool; idx++ ) {
```

Comment:
(style)  space prohibited after that open parenthesis '('
(style)  space prohibited before that close parenthesis ')'